src/main/resources/jupyter/jupyter-docker-compose.yaml

```diff
@@ -10,7 +10,7 @@ services:
     network_mode: host
     volumes:
       # shared with welder
-      - /work:/home/jupyter-user/notebooks
+      - /work:$(notebooksDir)
```

Comment:
This can be the EV that was exported: `${NOTEBOOKS_DIR}`.

This would also let you avoid needing to template `jupyter-docker-compose.yaml`.